projects/ngx-avatar/src/lib/sources/source.ts

```diff
@@ -20,5 +20,5 @@ export interface Source {
    * Gets the avatar that usually is a URL, but,
    * for example it can also be a string of initials from the name.
    */
-  getAvatar(size?: number): string;
+  getAvatar(size: number): string | null;
```

Comment:
Why not let this optional anymore?

Reply:
The argument is now non-null because it is always given by `AvatarComponent`. And it is always given because some sources require this argument and it is defined as non-null in `AvatarComponent` itself.

Reply:
I like the option to grab the original size, but if it's not used at this time there's really no reason to keep it here for now.